constantine/ethereum_bls_signatures.nim

```diff
@@ -510,3 +513,57 @@ func batch_verify*[Msg](pubkeys: openArray[PublicKey], messages: openarray[Msg],
   if verified:
     return cttEthBls_Success
   return cttEthBls_VerificationFailure
+
+func alloc_batch_sig_accumulator*(): ptr BatchSigAccumulator {.libPrefix: prefix_ffi.} =
+  ## Allocates using `alloc0` for a `BatchSigAccumulator`
+  ## so that it can remain an incomplete struct in the C header.
+  result = cast[ptr BatchSigAccumulator](allocHeapAligned(BatchSigAccumulator, alignment = 64))
```

Comment:
The casting is not needed by the way https://github.com/mratsim/constantine/blob/285aeb280f84b889195529bf821e2801423bcb08/constantine/platforms/allocs.nim#L101-L107

```suggestion
  result = allocHeapAligned(BatchSigAccumulator, alignment = 64)
```

Reply:
Heh, good catch!